Backend/routes/listings.js

```diff
@@ -9,12 +9,9 @@ const { sendNotification } = require("../services/notificationService");
 const NotificationType = require("../Enums/NotificationType.js");
 const logActivity = require("../middlewares/logActivity");
 const stripe = require("stripe")(process.env.STRIPE_SECRET_KEY);
-const { AuctionSystem } = require("../services/auctionSystem");
 
 const prisma = new PrismaClient();
-const auctionSystem = new AuctionSystem(prisma);
 const router = express.Router();
-router.use(verifyToken);
```

Comment:
any reason why we remove this? might have been easier to just have router use verifyToken so we don't have to add it everywhere right?

Reply:
Yeah, it was interfering with my run auction route which did not require it. While I was not using the verify token in the file, it was still being affected by it. One of the solutions i tired was removing the global verify token from all files and it worked, so I decided to use it on individual routes instead.